app/controllers/api/v1/application_users_controller.rb

```diff
@@ -92,6 +92,33 @@ def index
   end
 
   ###############################################################
+  # username
+  ###############################################################
+
+  api :GET, '/application_users/username/:username',
+      'Gets a single User with the specified username.'
+  description <<-EOS
+      All users of OpenStax have an associated User object.  This endpoint
+      allows querying additional data on a user when all that is known is the
+      User's username.
+
+      Admins (for Accounts only) are identified by the is_administrator boolean.
```

Comment:
Agree.  I've removed it and am referring to the model as the proper `ApplicationUser` now.  I think I was still attempting to figure out the nomenclature when I wrote that code. And probably still am :)